account/script-account.js

```diff
@@ -11,105 +11,212 @@ import {
 } from '@hashgraph/sdk';
 import dotenv from 'dotenv';
 
-async function scriptAccount() {
+async function createAccount() {
     // Read in environment variables from `.env` file in parent directory
     dotenv.config({ path: '../.env' });
 
     // Initialise the operator account
     const operatorIdStr = process.env.OPERATOR_ACCOUNT_ID;
-    const operatorKeyStr = process.env.OPERATOR_ACCOUNT_PRIVATE_KEY;
-    const account1PrivateKeyStr = process.env.ACCOUNT_1_PRIVATE_KEY;
-    const account2EvmAddress = process.env.ACCOUNT_2_EVM_ADDRESS;
-    if (!operatorIdStr || !operatorKeyStr || !account1PrivateKeyStr || !account2EvmAddress) {
-        throw new Error('Must set OPERATOR_ACCOUNT_ID, OPERATOR_ACCOUNT_PRIVATE_KEY, ACCOUNT_1_PRIVATE_KEY, and ACCOUNT_2_EVM_ADDRESS environment variables');
+    const operatorEcdsaKeyStr = process.env.OPERATOR_ACCOUNT_PRIVATE_KEY;
+
+    if (!operatorIdStr || !operatorKeyStr) {
+        throw new Error('Must set OPERATOR_ACCOUNT_ID, OPERATOR_ACCOUNT_PRIVATE_KEY');
     }
-    const operatorId = AccountId.fromString(operatorIdStr);
-    const operatorKey = PrivateKey.fromStringECDSA(operatorKeyStr);
+    const operatorAccountId = AccountId.fromString(operatorIdStr);
+    const operatorEcdsaKey = PrivateKey.fromStringECDSA(operatorKeyStr);
```

Comment:
This was created, but not used when initialising the `Client` object.
It appears to be a rename to include "ecdsa" without any functional difference.
Assume that it is OK to revert to original, so that it runs.